shell/models/management.cattle.io.gitreporestriction.js

```diff
@@ -0,0 +1,5 @@
+import SteveModel from '@shell/plugins/steve/steve-class';
+
+export default class Workspace extends SteveModel {
```

Comment:
I think this is the wrong class name for the model - should this not be `GitRepoRestriction` ?